jsonapi/jsonapi_utility.go

```diff
@@ -167,6 +171,11 @@ func JSONErrorResponse(obj interface{}, err error) error {
 			return errs.WithStack(ctx.Conflict(jsonErr))
 		}
 	default:
+		mutex.Lock()
```

Comment:
Instead of using the default client and locking the goroutine you could create a new client via `raven.New(dsn)` where `dsn := os.Getenv("SENTRY_DSN")`

Reply:
changed, raven-go docs need more love for sure.